scalikejdbc-config/src/test/resources/application-time-zone-conversion-enabled.conf

```diff
@@ -0,0 +1,2 @@
+scalikejdbc.global.timeZoneConversion.enabled=true
+scalikejdbc.global.timeZoneConversion.serverTimeZone=AST
```

Comment:
Isn't `scalikejdbc.global.timeZoneConversion.enabled` redundant?  An existence of `scalikejdbc.global.timeZoneConversion.serverTimeZone` can express that it's enabled.